### PR TITLE
add a run to shell_client with https

### PR DESCRIPTION
**Pull request by dothebart**

### CODE REVIEW

**Review by KVS85 — APPROVED** [review]
LGTM.

File: scripts/runFullTests.fish [hunk 1]
```diff
@@ -44,6 +44,8 @@ set ST "$ST""250,runSingleTest1 shell_server_aql 5 --testBuckets 6/5\n"
 set ST "$ST""250,runSingleTest1 server_http -\n"
 set ST "$ST""250,runSingleTest1 ssl_server 0 --testBuckets 2/0\n"
 set ST "$ST""250,runSingleTest1 ssl_server 1 --testBuckets 2/1\n"
+set ST "$ST""750,runSingleTest1 http --ssl false $EncryptionAtRest -\n"
+set ST "$ST""750,runSingleTest1 https --ssl true $EncryptionAtRest -\n"
```
Comment (jsteemann) [inline scripts/runFullTests.fish:48]:
```suggestion
set ST "$ST""750,runSingleTest1 shell_api http --ssl false $EncryptionAtRest -\n"
set ST "$ST""750,runSingleTest1 shell_api https --ssl true $EncryptionAtRest -\n"
```

**Review by fceller — APPROVED** [review]

**Review by jsteemann — APPROVED** [review]
LGTM

**Review by joerg84 — APPROVED** [review]

**Comment by KVS85** [comment]
Jenkins:

- [ ] 3.8:
  - [ ] Linux: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-linux-matrix.x86-64/3118/
  - [ ] macOS: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-mac-matrix/3121/
  - [ ] Windows: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-windows-matrix/3365/
- [ ] devel:
  - [ ] Linux: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-linux-matrix.x86-64/3117/
  - [ ] macOS: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-mac-matrix/3120/
  - [ ] Windows: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-windows-matrix/3364/

**Comment by KVS85** [comment]
- [ ] devel:
  - [ ] Linux: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-linux-matrix.x86-64/3160/
  - [ ] macOS: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-mac-matrix/3163/
  - [ ] Windows: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-windows-matrix/3407/

**Comment by KVS85** [comment]
- [ ] devel:
  - [ ] Linux: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-linux-matrix.x86-64/3161/
  - [ ] macOS: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-mac-matrix/3164/
  - [ ] Windows: https://jenkins.arangodb.biz:3456/job/arangodb-ANY-windows-matrix/3408/

**Pull request merged**